lib/base/journaldlogger.hpp

```diff
@@ -0,0 +1,41 @@
+/* Icinga 2 | (c) 2021 Icinga GmbH | GPLv2+ */
+
+#ifndef JOURNALDLOGGER_H
+#define JOURNALDLOGGER_H
+
+#include "config.h"
+#if !defined(_WIN32) && defined(HAVE_SYSTEMD)
+#include "base/i2-base.hpp"
```

Comment:
`base/i2-base.hpp` already includes `config.h` and all other files include that instead of `config.h`, so I think just moving this include before the `#if` would be the preferred way here.